### DISCUSSION TITLE
Mi practica de modulo CSS Terminada - Mazun

### DISCUSSION THREAD

**Discussion by LaMazorca** [Launch-X-Latam/MisionFrontEnd#276] [Show and tell]
Hola compañeros espero que tenga un grandioso día, les comparto mi página de la vacunación terminada, hasta lo que pide la practica, aún trabajo en las demás paginas, pero la actualizare, agradezco sus comentarios, a mi me gusto como quedo, pero creo que podría optimizar más el CSS, si me pudieran ayudar con eso.

[Repositorio](https://github.com/LaMazorca/Practica-CSS-LaunchX-LATAM-Vacunacion)

**Comment by doguedogue** [comment]
genial 🚀🚀🚀
